### PR TITLE
added prisma readme

### PR DESCRIPTION
**Pull request by xtoben22**

### CODE REVIEW

**Comment by coin96** [comment]
need to update prisma configuration and evm token mint feature

File: prisma/readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### DB migration
+
+```
+cargo run --bin prisma -- migrate dev
+```
```
Comment (coin96) [inline prisma/readme.md:5]: good
but you can leave your configuration detailed info

Comment (xtoben22) [inline prisma/readme.md:5]: it's better if leave more detailed configuration info

Comment (coin96) [reply prisma/readme.md:5]: yup
i agree
i mean that

Comment (xtoben22) [reply prisma/readme.md:5]: yup
no problem

**Pull request merged**